quickstarts/frontend/react.mdx

```diff
@@ -4,467 +4,408 @@ title: Integrate Hanko with React
 description: Learn how to quickly add authentication and user profile in your React app using Hanko.
 ---
 
+
+## Create a React application
+
+To setup our React Frontend we will use [vite](https://vite.dev/) as a build tool.
+
+Run the following command to [create a new Vite React application](https://vite.dev/guide/):
+
+<CodeGroup>
+```bash npm
+npm create vite@latest project-name -- --template react-ts
+```
+
+```bash pnpm
+pnpm create vite project-name --template react-ts
+```
+
+```bash bun
+bun create vite project-name --template react-ts
+```
+
+```bash yarn
+yarn create vite project-name --template react-ts
+``` 
+
+</CodeGroup>
+
+
+
 ## Install `@teamhanko/hanko-elements`
 
 Once you've initialized your React app, installing hanko-elements provides you with access to the prebuilt components: `hanko-auth` and `hanko-profile`.
 
+We will also install the `react-router-dom` , this will allow us to navigate trough pages / routes.
 <CodeGroup>
 ```bash npm
-npm install @teamhanko/hanko-elements
-```
+cd project-name
+npm install @teamhanko/hanko-elements react-router-dom
+``` 
 
 ```bash pnpm
-pnpm add @teamhanko/hanko-elements
+cd project-name
+pnpm add @teamhanko/hanko-elements react-router-dom
 ```
 
 ```bash bun
-bun add @teamhanko/hanko-elements
+cd project-name
+bun add @teamhanko/hanko-elements react-router-dom
 ```
 
 ```bash yarn
-yarn add @teamhanko/hanko-elements
+cd project-name
+yarn add @teamhanko/hanko-elements react-router-dom
 ```
 
 </CodeGroup>
 
+
+
+## Setup your Hanko project
+Go to the [Hanko console](https://cloud.hanko.io/) and [create a project for this application.](/setup-hanko-cloud)
+
+<Note>
+  During creation make sure to input the URL you will be developing on as the `APP URL`.
+  (Most likely http://localhost:5173/) \
+\
+  If you already have a project you can change the URL at:\
+  `'your project' > Settings > URLs > App URL`.
```

Comment:
The same as in the nextjs guide. I think this guide should be tailored to new users, that typically doesn't have an account already.